.gitlab-ci.yml

```diff
@@ -210,42 +240,62 @@ check:format:
   stage: check
   needs: []
 
+check:bundlewatch:
+  <<: *tmpl_branch
+  script:
+    - export CI_REPO_OWNER="$CI_PROJECT_NAMESPACE"
+    - export CI_REPO_NAME="$CI_PROJECT_NAME"
+    - export CI_BRANCH="$CI_COMMIT_BRANCH"
+    - yarn bundlewatch
+  stage: check
+  needs:
+    - build:ts:branch
+  dependencies:
+    - build:ts:branch
+
 ####################
 # stage deploy-dev #
 ####################
-deploy:sls:branch:
+
+.deploy:sls:
   <<: *tmpl_branch
   image: $INTEGRATION_DOCKER_IMAGE
   stage: deploy-dev
+  script:
+    - source ci/env-setup.sh
+    - yarn sls deploy --package .serverless --verbose
+
+deploy:sls:branch:
+  extends: .deploy:sls
+  rules:
```

Comment:
should we not use the `tmpl_branch` template? some of the jobs for the PR pipeline use it while others (like this one) just have the rule

Reply:
I have very mixed feelings about the tmp_branch template. We found it incredibly difficult to read and have it in our list of improvements to tackle. Maybe we could give them more descriptive names?

Reply:
yeah sounds good, we could either get rid of the templates entirely all use the same ones for every PR/dev/prod job, just to be able to tell them apart and not mix these up